internal/scheduler/schedulerobjects/schedulerobjects.proto

```diff
@@ -60,6 +60,13 @@ message QueueClusterResourceUsage {
     map<int32, ResourceList> resourcesByPriority = 4 [(gogoproto.nullable) = false];
 }
 
+// A collection of QueueClusterResourceUsage
+// This is only needed to brige the gap between the redis based scheduler and the new scheduler
+message ClusterResourceUsageReport {
```

Comment:
Instead of defining a new object, I'd prefer to just add the pool field to `QueueClusterResourceUsage`.

Reply:
I think we need a new object here- in this case because we need to store a collection (in my latest code this is a map) of `QueueClusterResourceUsage` not just a single one. I also want to add a timestamp here so that we can check if the data is out of date.  I agree that having to have a new object defined here isn't ideal but we should be able to retire it once we go to postgres.